### PR TITLE
Fixes a typo regarding unbound

### PR DESCRIPTION
**Pull request by asemoon**

### CODE REVIEW

File: src/target_resolver.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ class TargetResolver {
 
   _loadTarget(filepath) {
     try {
-      const obj = require(filepath);
+      const obj = require(`${filepath}`);
```
Comment (jiripospisil) [inline src/target_resolver.js:68]: What's the reason for this change? `filepath` is already guaranteed to be a string, isn't it?

Comment (jiripospisil) [reply src/target_resolver.js:68]: Ah, the warning. I see.

**Comment by asemoon** [comment]
@jiripospisil, what's the state of this PR? is it good to be merged or needs further changes?

**Comment by jiripospisil** [comment]
@asemoon It's ready to go. Thank you!
